CMakeLists.txt

```diff
@@ -122,9 +120,14 @@ endif()
 # Targets
 # ######################################################################################################################
 
+add_subdirectory(external/aos_core_common_cpp/external/aos_core_lib_cpp/src/common)
+add_subdirectory(external/aos_core_common_cpp/external/aos_core_lib_cpp/src/iam)
+add_subdirectory(external/api)
+add_subdirectory(external/aos_core_common_cpp/src)
```

Comment:
There is no cmake file in src. Is there?

Reply:
The cmake file is indeed present in the src folder for the common library.